app/views/conferences/_table.html.erb

```diff
@@ -1,61 +1,31 @@
-<table class="table-striped table-bordered table-condensed conferences">
-  <thead>
-  <tr>
-    <th> <%= sortable "Conference" %> </th>
-    <th> <%= sortable "Location" %> </th>
-    <th> <%= sortable "Dates" %> </th>
-    <th> Attendees </th>
-    <th> Tickets left </th>
-    <th> Lightning Talk Slots?</th>
-  </tr>
-  </thead>
-
-  <tbody>
-  <% conferences.where(round: 2).each do |conference| %>
-  <tr>
-    <td> <%= link_to conference.name, [namespace, conference] %></td>
-    <td> <%= conference.location %></td>
-    <td> <%= conference.date_range %></td>
-    <td> <%= conference.attendees.any? ? links_to_attendances(conference).join(', ').html_safe : '-' %></td>
-    <td> <%= conference.tickets_left %></td>
-    <td><%= conference.lightningtalkslots? ? "yes" : "no" %></td>
-  </tr>
-      <% end %>
-    </tbody>
-</table>
-
-<br>
-<p><%= link_to "View as Calendar", calendar_path %>
-</p>
-
-<p><%= link_to "Show/Hide First Round Conferences", "#", onclick: "$('.first-conferences').toggle()" %></p>
-
-<div class="first-conferences" style='display:none;'>
-<h1 class=" header">Conferences (1st round)</h1>
-
+<h1 class="header">Conferences</h1>
```

Comment:
The excuse is that I thought we were going to get rid of slim. The real reason is that I found slim very confusing, so I always convert to and from erb, and that is not time I feel is well spent. Can you live withe erb here?